### PR TITLE
9349 Make switch widget accessible

### PR DESCRIPTION
**Pull request by johnatawnclementawn**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Improve a user's ability to navigate through the switch widget and its configuration using a screen reader/keyboard. Allow the user to substitute and keypress for a click event. Keypress feature also included in #9402 

### Issues Solved

#9349 

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: arches/app/templates/views/components/widgets/switch.htm [hunk 1]
```diff
@@ -3,7 +3,8 @@
 
 {% block form %}
 <div class="toggle-container">
-    <span class="switch switch-small switch-widget" data-bind="css: {'on': getvalue() === true, 'null': getvalue() === null, 'disabled': disabled }, click: setvalue"><small></small></span>
+    <span class="switch switch-small switch-widget" data-bind="css: {'on': getvalue() === true, 'null': getvalue() === null, 'disabled': disabled }, 
+          onEnterkeyClick, onSpaceClick, click: setvalue, attr: {'aria-checked': getvalue}" tabindex="0" role="switch"><small></small></span>
```
Comment (apeters) [inline arches/app/templates/views/components/widgets/switch.htm:7]: add `aria-describedby` that points to the subtitle span below

File: arches/app/media/css/arches.scss [hunk 2]
```diff
@@ -217,26 +217,14 @@ dl.inline-flex dd {
     color: #888;
 }
 
-.switch.switch-small.switch-widget {
-    width: 35px;
-}
-
 .iiif-image-tool-slider .switch.switch-small.switch-widget {
```
Comment (apeters) [inline arches/app/media/css/arches.scss:220]: I think we'll want to move this to the _switch.scss

File: arches/app/media/css/arches.scss [hunk 3]
```diff
@@ -217,26 +217,14 @@ dl.inline-flex dd {
...
     width: 25px;
 }
 
-.switch.switch-widget.on>small {
-    left: 22px;
-}
-
 .iiif-image-tool-slider .switch.switch-widget.on>small {
```
Comment (apeters) [inline arches/app/media/css/arches.scss:224]: I think we'll want to move this to the _switch.scss

**Review by apeters — COMMENTED** [review]
I think we'll need to move all the css that is tagged `.switch` and `.switch-small` over to the _switch.scss file

File: arches/app/templates/views/components/widgets/switch.htm (hunk 1 above)
Comment (johnatawnclementawn) [reply arches/app/templates/views/components/widgets/switch.htm]: I'm experiencing `aria-describedby` breaking the keyboard click handlers and not letting user tab out of the switch element. How do you feel about concatenating the `label` & `localizedSubtitle`?

Comment (apeters) [reply arches/app/templates/views/components/widgets/switch.htm]: Hmm, that's weird.  I guess that's fine to concatenate those strings.

**Review by apeters — APPROVED** [review]
looks good

**Pull request merged**